src/classes/BDI_AdditionalObjectServiceTest.cls

```diff
@@ -0,0 +1,1275 @@
+/*
+    Copyright (c) 2014 Salesforce.org
```

Comment:
These copyright years are all over the place; presumably they should be 2019 for new components.